src/LoginDialog/OidcUtilities.ts

```diff
@@ -25,9 +25,9 @@ export const generateOidcLoginRedirectUrl = (baseRedirectUrl: string, cId: strin
         'scope=openid+email+profile'
     ].join('&');
     
-    return `${baseRedirectUrl}?${oidcParameters}`
+    return `${baseRedirectUrl}&${oidcParameters}`
```

Comment:
i'm thinking if some extra checks are needed here to address corner cases

Reply:
If you're certain that you're returing a url, you can create a URL and append params to it.
Here's a fiddle of what I'm referring to:
https://jsfiddle.net/u1p8fnbL/

Reply:
Thanks, will do